controller/scheduler/swarm.py

```diff
@@ -1,11 +1,21 @@
 import re
 import time
-
+import logging
+import os
 from django.conf import settings
 from docker import Client
 
 from .states import JobState
 
+logger = logging.getLogger('api')
+
+
+def log_event(appid, msg, level=logging.INFO):
```

Comment:
let's change any calls here to `app.log()` instead so we're not maintaining duplicate code